src/test/lib/DataTest.cpp

```diff
@@ -788,6 +788,113 @@ QuicTestClientDisconnect(
     }
 }
 
+void
+QuicTestStatelessResetKey(
+    )
+{
+    //
+    // If the listener is stopped at the same time the server side of the
+    // connection is silently closed, then the UDP binding will also be cleaned
+    // up. This means the endpoint will no longer send Stateless Reset packets
+    // back to the client as it continues to receive the client's UDP packets.
+    //
```

Comment:
This comment should be updated to reflect what exactly is being tested here; essentially, that by changing the stateless reset key, the stateless reset packets the client receives after the server side is shut down no longer match, eventually resulting in a timeout on the client instead of an abort.